test/data/Anode_Graphite.dat

```diff
@@ -0,0 +1,108 @@
+*******************************************************************************************************************************
+*
+*Enthalpy and Entropy values in dependence of stoichiometry of Lithium content in Electrodes
+*
+*
+*Electrode material:
+*A123-graphite
+*
+*Source:
+*
+*dH: from Delacourt yet unpublished 2011
+*dS: Reynier 2004 JElchemSoc - Thermodynamics of Lithium Intercalation into Graphites and Disordered Carbons
+*original range: 0.01 - 0.8
+* fit of original data
+*
+*Author: Wolfgang Mielke 21.03.2011
+*File format: '*' starts a comment line. Values may be separated by one or multiple CR, TAB or SPACE.
+*
+*columns:		
+*Stoichiometry x [], dH [J/mol],  dS [J/mol/K]		

```

Comment:
By default, units in Cantera should be J/kmol and J/kmol/K, with the ability to specify alternate units on a case-specific basis.